session.go

```diff
@@ -131,9 +129,22 @@ func startSession(conn net.Conn, windowSize int, maxPadding int, ackOnFirst bool
 	atomic.AddInt64(&openSessions, 1)
 	ops.Go(s.sendLoop)
 	ops.Go(s.recvLoop)
+	if clientInitMsg != nil {
+		s.sendClientInitMsg(clientInitMsg)
```

Comment:
This is an interesting change. The original design was meant to coalesce the first application-level data with the client init message to further obfuscate it. I think your approach works too though, just has slightly more overhead. The important thing is that it works for you in practice.